src/Controller/ResetPasswordController.php

```diff
@@ -90,33 +89,27 @@ public function reset(Request $request, UserPasswordEncoderInterface $passwordEn
         $user = $this->resetPasswordHelper->validateTokenAndFetchUser($token);
 
         //Reset password after token verified
-        $form = $this->createForm(PasswordResettingFormType::class);
+        $form = $this->createForm(ResetPasswordResetFormType::class);
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
             // A ResetPasswordToken should be used only once, remove it.
             $this->resetPasswordHelper->removeResetRequest($token);
 
             // Encode the plain password, and set it.
-            //@TODO Encode password expects a UserInterface, we are only guaranteeing an object
             $encodedPassword = $passwordEncoder->encodePassword(
                 $user,
                 $form->get('plainPassword')->getData()
             );
 
-            /**
-             * We are assuming the user object HAS a setPassword method..
-             * @TODO handle this if it doesnt/sets password using different method.
-             * @psalm-suppress UndefinedInterfaceMethod
-             */
             $user->setPassword($encodedPassword);
             $this->getDoctrine()->getManager()->flush();
 
-            // TODO: please check the login route | CHANGE ROUTE, APP_FORGOT_PASSWORD_REQUEST USED IN DEVELOPMENT.
-            return $this->redirectToRoute('app_forgot_password_request');
+            // @TODO: please check the login route
+            return $this->redirectToRoute('app_home');
```

Comment:
Hmm, yea. We've never done it before in Maker, but I feel like we should ask them which route to redirect to after a successful reset (and validate their response).

Reply:
I toyed around with this earlier. Just from personal experience, when using maker in a new project, I often don't know what the route will be (if it doesn't exist yet) at the time of running maker commands..

But I think we could have an optional question in there for this. Perhaps default/place-holder for the question  is `app_home`.

Reply:
done

maker has a required arg thats populated via `ask`. default is `app_home`.